Configuration/ProcessModifiers/python/alpakaValidationParticleFlow_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is for enabling alpaka modules
+
+alpakaValidationParticleFlow = cms.Modifier()
```

Comment:
I would prefer that we do not introduce yet another modifier, and use the `alpaka` modifier instead.

Reply:
I was using this to also enable the Legacy clustering (as opposed to only alpaka) as well as enable the Alpaka vs Legacy cluster-level comparison. I am trying to think of an elegant solution around this, or would it perhaps be fine to let `alpaka` do these modifications (run Alpaka & Legacy clustering) to HCAL-only workflows since it is probably just used for benchmarking and testing?

Reply:
IMHO there should be two sequences
  - one for HCAL-only
  - one for HCAL-only validation

The former should run the HCAL-only reconstruction (and DQM)
  - by default using the legacy CPU modules
  - if the `gpu` modifier is present, using the CUDA modules
  - if the `alpaka` modifier is present, using the Alapaka modules

The second should to the same as the first one (depending on the modifier), but in addition run a second copy of the legacy CPU modules, and the DQM comparison.

Then, cmsDriver can pick the first or second sequence depending on the command line options.

Reply:
So I guess to do this, introducing the HCAL-only validation sequence you described would be done in `Configuration/StandardSequences/python/Reconstruction_cff.py`?